ldproxy/src/main.rs

```diff
@@ -140,7 +142,7 @@ fn args() -> Result<Vec<String>> {
             }
             // otherwise just add the argument as normal
             else {
-                result.push(arg);
+                result.push(arg.to_owned());
```

Comment:
This now adds just the rest of the arg after the `@`-sign which isn't correct. The gcc docs say that if the response file doesn't exist the argument is interpreted as is, so with the `@` prefix.

I don't think there is such a case where a `@<...>` isn't supposed to be a response file, but its not as per docs nonetheless.

Reply:
Indeed - thanks & fixed.